### ISSUE TITLE
found no layout file for "HTML" for kind "home"

### ISSUE THREAD

**Issue by chansters** [jpanther/congo#530]
### What happened?

I'm a Hugo newbie and I'm having trouble getting this theme up and running. I've followed the excellent and detailed documentation on the theme website but I keep this error when building the site:

> Start building sites …
hugo v0.111.3+extended linux/amd64 BuildDate=unknown
WARN 2023/04/09 07:42:09 found no layout file for "HTML" for kind "home": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2023/04/09 07:42:09 found no layout file for "HTML" for kind "taxonomy": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2023/04/09 07:42:09 found no layout file for "HTML" for kind "taxonomy": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2023/04/09 07:42:09 found no layout file for "JSON" for kind "home": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.

My homepage layout is set to "page" and I have a "_index.md" file in the /content directory. But I'm getting this error message when trying to load the site:

> Page Not Found

I installed the theme using git. 

### Theme version

v2.5.4

### Hugo version

v0.111.3+extended

### What browsers are you seeing the problem on?

Chrome

**Comment by chansters** [comment]
Update : I decided to delete the site and start over and install the theme using the Hugo module method instead.  Everything was working fine and Hugo was able to build a blank site with the theme.  

However, I ran into the same issue once I started copying the .toml files into 'config/_default/'.  I downloaded the fresh config files from the [link](https://minhaskamal.github.io/DownGit/#/home?url=https://github.com/jpanther/congo/tree/stable/config/_default) in the documentation.  I also deleted the config.toml from the root directory as instructed.
